### ISSUE TITLE
Incorrect place name in positron

### ISSUE THREAD

**Issue by nickdos** [CartoDB/CartoDB-basemaps#53]
The following tile has `Lord Howe Island` incorrectly labelled as `Norfolk Island`.

http://a.basemaps.cartocdn.com/light_all/5/30/18.png

Better illustrated in this screen shot:

![image](https://cloud.githubusercontent.com/assets/532845/13206038/2c7e8560-d94a-11e5-8ace-4acae4cde899.png)

Please correct this. Note OSM correctly labels these islands.

**Comment by pnorman** [comment]
This isn't coming from [aries-magic-list](https://github.com/stamen/aries-magic-list/) or fixed in the develop branch, so it looks like a Natural Earth issue

**Comment by almccon** [comment]
Confirmed it's an error in Natural Earth's `ne_10m_admin_1_states_provinces_shp` file. Here are the `name` labels in that file, rendered in QGIS:

<img width="783" alt="screen shot 2016-03-02 at 2 mar 5 26 23" src="https://cloud.githubusercontent.com/assets/1212178/13461729/36373490-e038-11e5-9777-4338b4f5a359.png">

**Comment by almccon** [comment]
I won't have time to fix this right away, but if anyone else wants to upload a modified version of that shapefile, I can merge it into the Stamen fork of Natural Earth https://github.com/stamen/natural-earth-vector.

**Comment by pnorman** [comment]
I had a look at the upstream natural earth, but couldn't find how to contribute, and the age of unmerged PRs isn't encouraging.

**Comment by almccon** [comment]
@pnorman Yes, contributing to upstream is hard, because most of the files in github are actually derived products from an ESRI Geodatabase that lives on someone's machine somewhere. So I personally wouldn't know how to fix the source issues anyway.

But whenever I find errors where I can fix the resulting shapefile (by editing it in QGIS) I push fixed zipped versions to the Stamen branch, and source those for the rendering.

See https://github.com/stamen/toner-carto/blob/master/Makefile#L260-L263 and https://github.com/stamen/toner-carto/blob/master/Makefile#L295-L303 in Toner, where some of the Natural Earth is sourced from naturalearthdata.com, and some of it is sourced from github from Stamen's fork.

I think we might have done that originally for some of the CartoDB basemap layers, but looking at [datasets.txt](https://github.com/CartoDB/CartoDB-basemaps/blob/master/datasets.txt) it seems that currently it's only coming from naturalearthdata.com

**Comment by nickdos** [comment]
Our users have reported another incorrect label, similar to the one above.

"Geographe Bay" in Western Australia, is incorrectly placed.

CartoDB has it just below Perth but its actually a few hundred km further south. See 2 screen shots:

![image](https://cloud.githubusercontent.com/assets/532845/19224167/f285bd0a-8ecb-11e6-99fd-a1c2a9672ba5.png)

![geographe-bay-osm](https://cloud.githubusercontent.com/assets/532845/19224182/371d5964-8ecc-11e6-81c4-db1c8363d5b1.png)

 (orange dot OSM location marker)
